pkgs/development/python-modules/pyobjc-framework-Cocoa/tests.patch

```diff
@@ -0,0 +1,166 @@
+diff --git a/PyObjCTest/test_cffilesecurity.py b/PyObjCTest/test_cffilesecurity.py
+index f0121d8ac..e77000b37 100644
+--- a/PyObjCTest/test_cffilesecurity.py
++++ b/PyObjCTest/test_cffilesecurity.py
+@@ -21,16 +21,6 @@ class TestCFFileSecurity(TestCase):
+     def testConstants(self):
+         self.fail("kCFFileSecurityRemoveACL")
+
+-    @min_os_level("10.8")
+-    @expectedFailure
+-    def testConstants10_8(self):
```

Comment:
Also here I would rather ignore the tests with pytestCheckHook as a whole rather than patching, because patching causes too much maintenance overhead.